docs/modules/operation/pages/deep-dive/provisioning/auto-discovery.adoc

```diff
@@ -91,33 +119,33 @@ In addition to the configuration above, there are some optional settings you can
 
 | initial-sleep-time
 | 30000 ms (30 seconds)
-| The time, in milliseconds, before the discovery process begins after OpenNMS has started.
+| The time, in milliseconds, before the discovery process begins when OpenNMS has started.
```

Comment:
Since the next line says "This delay allows Horizon to fully start before generating new events," I think you're right; "after" might be more appropriate here. I'll update.